File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/GaitAnalysisComputerBasedFormViewModel.ts
//$CB3D5809
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { GaitAnalysisComputerBasedForm } from "NebulaClient/Model/AtlasClientModel";
import { Type } from 'NebulaClient/ClassTransformer';

export class GaitAnalysisComputerBasedFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.GaitAnalysisComputerBasedFormViewModel, Core';
    @Type(() => GaitAnalysisComputerBasedForm)
    public _GaitAnalysisComputerBasedForm: GaitAnalysisComputerBasedForm = new GaitAnalysisComputerBasedForm();
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/GaitAnalysisComputerBasedFormViewModel.cs
//$A07203FF
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;
using Core.Controllers;

namespace Core.Controllers
{
    public partial class GaitAnalysisComputerBasedFormServiceController
    {
    }
}

namespace Core.Models
{
    public partial class GaitAnalysisComputerBasedFormViewModel: BaseAdditionalInfoFormViewModel
    {
        public override BaseAdditionalInfo AddViewModelToContext(TTObjectContext objectContext)
        {
            var gaitAnalysisComputerBasedFormService = new GaitAnalysisComputerBasedFormServiceController();
            gaitAnalysisComputerBasedFormService.GaitAnalysisComputerBasedFormInternal(this, objectContext);
            return (GaitAnalysisComputerBasedForm)objectContext.GetObject(this._GaitAnalysisComputerBasedForm.ObjectID, "GaitAnalysisComputerBasedForm");
        }
    }
}
